android/src/main/java/in/juspay/hypersdkreact/HyperSdkReactModule.java

```diff
@@ -248,6 +261,31 @@ public void onEvent(JSONObject data, JuspayResponseHandler handler) {
                         }
                         sendEventToJS(data);
                     }
+
+                    @Nullable
+                    @Override
+                    public View getMerchantView(ViewGroup viewGroup, MerchantViewType merchantViewType) {
+                        if (rim == null) {
+                            return super.getMerchantView(viewGroup, merchantViewType);
+                        } else {
+                            ReactRootView rrv = new ReactRootView(getCurrentActivity());
+                            switch (merchantViewType) {
+                                case HEADER:
+                                    rrv.startReactApplication(rim, "JuspayHeader");
```

Comment:
As these constants are used in native code, we can expose them from here itself to js/ts to merchant